day_one_part_one.py

```diff
@@ -8,20 +8,15 @@
 
 lines = test_input.splitlines()
 number = ""
-list = []
-total = 0
+numbers = []
 
 for line in lines:
   for char in line:
     if char.isdigit():
       number += char
-  list.append(int(number[0] + (number[-1])))
+  numbers.append(int(number[0] + (number[-1])))
   number = ""
 
-for entry in list:
-  total += int(entry)
+total = sum(numbers)
```

Comment:
You can call sum numbers on an iterable (anything you can do `for bleh in iterable`
It's super duper fast for some reason!